noir-projects/aztec-nr/aztec/src/context/public_context.nr

```diff
@@ -185,10 +186,9 @@ impl PublicContext {
 fn gas_for_call(user_gas: GasOpts) -> [Field; 2] {
     // It's ok to use the max possible gas here, because the gas will be
     // capped by the gas left in the (STATIC)CALL instruction.
-    let MAX_POSSIBLE_FIELD: Field = 0 - 1;
     [
-        user_gas.l2_gas.unwrap_or(MAX_POSSIBLE_FIELD),
-        user_gas.da_gas.unwrap_or(MAX_POSSIBLE_FIELD)
+        user_gas.l2_gas.unwrap_or(MAX_FIELD_VALUE),
+        user_gas.da_gas.unwrap_or(MAX_FIELD_VALUE)
```

Comment:
@ AVM team - I think this change triggered the code owners review. I've added a max field value const to unify/tidy where we use it across the repo, including in the public context.